content/events/gcc2023/header.md

```diff
@@ -1,47 +1,21 @@
 
 <div class="trim-p">
 
-<!-- https://commons.wikimedia.org/wiki/File:The_heart_reef,_part_of_the_Great_Barrier_Reef_near_Airlie_Beach,_Whitsunday_Islands,_Queensland.jpg -->
-[![gcc2023](/images/events/gcc2023/au-heart-island.png)](/events/gcc2023/)
+[![gcc2023](/images/events/gcc2023/au-gcc-banner.png)](/events/gcc2023/)
 
 </div>
 <div class="linkbox-horizontal trim-p">
 
 [Home](/events/gcc2023/) |
-[**Abstracts**](/events/gcc2023/abstracts/) |
+[Abstracts](/events/gcc2023/abstracts/) |
```

Comment:
Given how many links are here, I like highlighting 1 or 2 as a way to draw users's attention to the links that are most time-pertinent... But that's just my opinion.

Reply:
It is a bit weird/confusing though, because that usually infers which page is currently active!